src/functions/GaussExp.h

```diff
@@ -66,7 +66,7 @@ class GaussExp: public RepresentableFunction<D> {
     void operator*=(double d) { this->multInPlace(d); }
 
     double getScreening() const { return screening; }
-    double getExp(int i) const { return this->funcs[i]->getExp(); }
+    auto getExp(int i) const { return this->funcs[i]->getExp(); }
```

Comment:
Mmmm... I'm not so sure about return type deduction. This makes is pretty hard to understand what's going on for people reading the code, unless they use an IDE (which I guess is _not_ the majority of people working on MRCPP and MRChem) It might come in handy for templated functions, where it's hard or impossible to use the trailing return type declaration...

Reply:
agreed